src/mixer/basetrackplayer.cpp

```diff
@@ -618,6 +653,20 @@ void BaseTrackPlayerImpl::slotWaveformZoomSetDefault(double pressed) {
     m_pWaveformZoom->set(defaultZoom);
 }
 
+void BaseTrackPlayerImpl::slotShiftCues(double milliseconds) {
+    if (m_pLoadedTrack == nullptr) {
```

Comment:
Prefer `if (!m_pLoadedTrack)` for brevity. And this also works for options, not only pointers.